metadata.json

```diff
@@ -1,15 +1,21 @@
 {
   "name": "puppet-misp",
-  "version": "0.2.0",
+  "version": "0.2.3-rc0",
```

Comment:
bump it to the next version. I guess that will be 0.2.2 (thats mentioned in the CHANGELOG.md)? Please don't add the -rc0.